cmaes/cma.py

```diff
@@ -129,6 +129,7 @@ def ask(self) -> Tuple[np.ndarray, np.ndarray]:
 
     def _sample_solution(self) -> Tuple[np.ndarray, np.ndarray]:
         if self._B is None or self._D is None:
+            self._C = (self._C + self.C.T) / 2
```

Comment:
it seems mypy is failed.
https://github.com/c-bata/cmaes/pull/5/checks?check_run_id=418765134

```suggestion
            self._C = (self._C + self._C.T) / 2
```